woodwork/column_accessor.py

```diff
@@ -434,6 +484,59 @@ def box_plot_dict(
         """
         return _get_box_plot_info_for_column(
             self._series,
+            method="box_plot",
+            quantiles=quantiles,
+            include_indices_and_values=include_indices_and_values,
+            ignore_zeros=ignore_zeros,
+        )
+
+    @_check_column_schema
+    def medcouple_dict(
```

Comment:
This is added as a convenience function to match `box_plot_dict`. Ideally `box_plot_dict` would have been called `iqr_dict` to avoid confusion about the outlier calculation method (IQR aka Box Plot) vs the way we display the outlier information to the user (Box and Whiskers Plot), since both `box_plot_dict` and `medcouple_dict` generate information for a Box and Whiskers Plot. We will maintain the function name of `box_plot_dict` to avoid a sizable breaking change.